libminifi/include/provenance/Provenance.h

```diff
@@ -360,7 +361,7 @@ class ProvenanceEventRecord : public core::SerializableComponent {
   bool DeSerialize(const std::shared_ptr<core::SerializableComponent> &repo);
 
   uint64_t getEventTime(const uint8_t *buffer, const size_t bufferSize) {
-    int size = bufferSize > 72 ? 72 : bufferSize;
+    int size = gsl::narrow<int>(bufferSize > 72 ? 72 : bufferSize);
```

Comment:
I think this is simply an std::min written in complex form :)

Reply:
true -- I've replaced it with `min`